client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -49,7 +48,7 @@ function getPurchases( props ) {
 	return props.receipt.data.purchases;
 }
 
-var CheckoutThankYou = React.createClass( {
+const CheckoutThankYou = React.createClass( {
```

Comment:
@scruffian I don't think we should be using `const` here until the entire module uses ES6.